libraries.json

```diff
@@ -507,17 +502,19 @@
   {
     "github": "StarryBlueSky/Penicillin",
     "category": "Social",
-    "maven": "https://dl.bintray.com/nephyproject/stable/blue/starry/penicillin/"
+    "maven": "https://repo1.maven.org/maven2/blue/starry/penicillin/"
   },
   {
     "github": "StarryBlueSky/Json.kt",
     "category": "Serialization",
-    "maven": "https://dl.bintray.com/nephyproject/stable/blue/starry/jsonkt/"
+    "maven": "https://repo1.maven.org/maven2/blue/starry/jsonkt/"
   },
   {
     "github": "p7s1-ctf/SevenFacette",
     "category": "Test",
-    "maven": "https://dl.bintray.com/p7s1qa/sevenfacette-jvm/de/p7s1/qa/sevenfacette/core/"
+    "maven": "https://maven.pkg.jetbrains.space/sevenfacette/p/sevenfacette/sevenfacette",
+    "username": "85dcdf4c-d6cd-486f-85d3-654904e1cf22",
+    "password": "365171aab3c77e6aa2016b73c29ba738280dd96e94dddfc15e45ef65ffbcec5b"
```

Comment:
what is it? where it's used? why it's added?

Reply:
This is credentials to get from private maven repo https://github.com/munichbughunter/SevenFacette#jetbrains-space

Reply:
1. fetcher not use this fields
2. it's secrets that should not be here
3. fetcher not have auth logic - remove this lib

Reply:
removed